build.rs

```diff
@@ -19,7 +19,6 @@ fn main() {
         serde_json::from_str(drivers_json.as_str()).expect("Failed to parse drivers.json");
 
     let dst = cmake::Config::new("acquire-common")
-        .target("acquire-common")
```

Comment:
This does indeed set the [target triple](https://docs.rs/cmake/latest/cmake/struct.Config.html#method.target), so we might as well get rid of it.